quinn-udp/src/windows.rs

```diff
@@ -53,6 +90,34 @@ impl UdpSocketState {
             if rc == -1 {
                 return Err(io::Error::last_os_error());
             }
+
+            if wsa_recvmsg_ptr.is_some() {
+                let rc = unsafe {
+                    WinSock::setsockopt(
```

Comment:
Idea: add a helper function as done in unix.rs

Reply:
That would be welcome for readability.

Reply:
ok i will do it